apps/volontulo/views/offers.py

```diff
@@ -483,13 +483,12 @@ def post(request, slug, id_):  # pylint: disable=unused-argument
 
             offer.volunteers.add(user)
             offer.save()
-
             send_mail(
                 request,
                 'offer_application',
                 [
-                    user.email,
-                    request.POST.get('email'),
+                    up.user.email
+                    for up in offer.organization.userprofiles.all()
```

Comment:
I prefer not to use shortcuts as a name of variables this improve readability in code, suggest to change this variable name from ``` up ``` to ``` user_profile ```